operator/controllers/utils.go

```diff
@@ -139,7 +139,7 @@ func CheckIndex(ctx context.Context, indexName string) (bool, error) {
 	resp, err := client.Do(req)
 	if err != nil {
 		logger.Error(err, "failed to fetch log indexes")
-		return false, err
+		return false, nil
```

Comment:
why are we ignoring this error?

Reply:
oh I forgot to revert this back, I will send a fix for this tommorow